src/main/java/com/possible/dhis2int/Properties.java

```diff
@@ -5,43 +5,46 @@
 
 @Configuration
 public class Properties {
-	
+
 	@Value("${bahmni.login.url}")
 	public String bahmniLoginUrl;
-	
+
 	@Value("${openmrs.root.url}")
 	public String openmrsRootUrl;
-	
+
 	@Value("${reports.json}")
 	public String reportsJson;
-	
+
 	@Value("${reports.url}")
 	public String reportsUrl;
-	
+
 	@Value("${dhis.config.directory}")
 	public String dhisConfigDirectory;
-	
+
 	@Value("${dhis.url}")
 	public String dhisUrl;
-	
+
 	@Value("${openmrs.db.url}")
 	public String openmrsDBUrl;
-	
+
 	@Value("${openelis.db.url}")
 	public String openelisDBUrl;
-	
+
 	@Value("${dhis.password}")
 	public String dhisPassword;
-	
+
 	@Value("${dhis.user}")
 	public String dhisUser;
-	
+
 	@Value("${log4j.config.file}")
 	public String log4jConfigFile;
-	
+
 	@Value("${submission.audit.folder}")
 	public String submissionAuditFolder;
-	
+
 	@Value("${dhis.imam.dataset.id}")
 	public String dhisImamDataSetId;
+
+	@Value("${dhis.familyPlanning.dataset.id}")
+	public String dhisFamilyPlanningDataSetId;
```

Comment:
@suruchee  User standard formatters as I see lots of format changes in other files as well.